app/jobs/scheduled/update_animated_uploads.rb

```diff
@@ -13,7 +13,7 @@ def execute(args)
         .limit(MAX_PROCESSED_GIF_IMAGES)
         .each do |upload|
         uri = Discourse.store.path_for(upload) || upload.url
-        upload.update!(animated: FastImage.animated?(uri))
+        upload.update_columns(animated: FastImage.animated?(uri))
```

Comment:
`update_columns` doesn't touch the `updated_at` timestamp like `save!` did. For consistency it would be good to manually set it to `Time.current`.

Reply:
I would not like to mess with the `updated_at` field. What do you say if I use `upload.save(validate: false)` to let Rails do it for me, but also skip validations? 😊

Reply:
Either is cool! 🙂